README.md

```diff
@@ -1,70 +1,210 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<!-- TABLE OF CONTENTS -->
 
-## Available Scripts
+# 📗 Table of Contents
 
-In the project directory, you can run:
+- [📖 Bookstore Project](#bookstore-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#getting-started)
+    - [Install](#-install)
+    - [Usage](#-usage)
+    - [Aviable Scripts](#aviable-scripts)
```

Comment:
- [ ] Clicking on this link should redirect the user to the appropriate part of the readme file. Kindly fix the link  so that it works for a professional readme file